core/src/main/java/io/grpc/inprocess/InProcessChannelBuilder.java

```diff
@@ -44,7 +45,9 @@
  *
  * <p>The channel is intended to be fully-featured, high performance, and useful in testing.
  */
-public class InProcessChannelBuilder extends AbstractChannelBuilder<InProcessChannelBuilder> {
+@Internal
```

Comment:
I was under the assumption that all of the builders were to be exposed as public indirectly via public bootstrap methods, like @louiscryan did for android in https://github.com/grpc/grpc-java/pull/902.